mempool/src/tests/integration_tests.rs

```diff
@@ -29,7 +29,7 @@ fn single_node_test() {
         other_peer_network_id,
         PeerRole::Validator,
         ConnectionOrigin::Outbound,
-        Some(&[ProtocolId::MempoolDirectSend]),
+        &[],
```

Comment:
Silly question: why did we remove these? Is it because they're the same as the default protocols?

Reply:
Yeah, it's the same as the default protocols.  Then when they are changed we don't have to go through and be explicit.